src/construction.cpp

```diff
@@ -313,14 +337,19 @@ void construction_menu()
 
             if( !constructs.empty() ) {
                 std::string current_desc = constructs[select];
+                mvwprintz( w_con, w_height - 5, ( w_list_width + w_list_x0 + 2 ), c_white,
```

Comment:
```trim_and_print()``` here and below is a good way to avoid glitches. You can use variable ```available_window_width``` defined above as ```width``` parameter of function.